bcda/web/api_test.go

```diff
@@ -1252,56 +1246,6 @@ func (s *APITestSuite) TestServeData() {
 	assert.Contains(s.T(), s.rr.Body.String(), `{"resourceType": "Bundle", "total": 33, "entry": [{"resource": {"status": "active", "diagnosis": [{"diagnosisCodeableConcept": {"coding": [{"system": "http://hl7.org/fhir/sid/icd-9-cm", "code": "2113"}]},`)
 }
 
-func (s *APITestSuite) TestAuthTokenMissingAuthHeader() {
```

Comment:
These are good to delete because of duplication with [tests in auth/api.go](https://github.com/CMSgov/bcda-app/blob/96f3e97b18bdcec78c71732cf65748e624992193/bcda/auth/api_test.go#L69-L110)